### PR TITLE
update ROADMAP.md and README.md

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

**Comment by amylindburg** [comment]
LGTM!

If you are able to paste in links to GitHub PRs/issues -- for clarity -- that would be cool.

**Comment by aluzzardi** [comment]
LGTM

File: ROADMAP.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ In the future, we want to allow third party tools to be plugged into Docker Swar
...
 The work has started on defining the "cluster" interface in [#393](https://github.com/docker/swarm/issues/393)
-qThe first integration will be Mesos but other tools will arrive after, like kubernetes.
+
+The first integration will be Mesos but other tools will arrive after, like kubernetes.
 
 * [ ] Cluster Drivers
+
+Project Planning
+================
+* [0.2.0 Milestone](https://github.com/docker/swarm/wiki/0.2.0-Milestone-Project-Page)
```
Comment (samalba) [inline ROADMAP.md]: We should make it clear that each milestone (for instance 0.2.0) maps to a platform release (Docker 1.6)

File: ROADMAP.md [hunk 2]
```diff
@@ -33,6 +33,12 @@ In the future, we want to allow third party tools to be plugged into Docker Swar
 of much bigger clusters.
 
 The work has started on defining the "cluster" interface in [#393](https://github.com/docker/swarm/issues/393)
-qThe first integration will be Mesos but other tools will arrive after, like kubernetes.
+
+The first integration will be Mesos but other tools will arrive after, like kubernetes.
 
 * [ ] Cluster Drivers
+
+Project Planning
```
Comment (samalba) [inline ROADMAP.md]: s/Project Planning/Project Milestones/

Comment (vieux) [inline ROADMAP.md]: We discussed that with @amylindburg, if we put `Project Milestones` I expected to see the github milestones linked below, not the project pages.

File: ROADMAP.md (hunk 1 above)
Comment (amylindburg) [inline ROADMAP.md]: How about just replicating the chart on the wiki home page, which maps the Platform Milestones to the Swarm Milestones, and clearly defines the Project Page:

| Platform Version | Swarm Milestone | Project Page | 
|------------------|-----------------|--------------|
| Docker 1.6.0 | [0.2.0](https://github.com/docker/swarm/milestones/0.2.0) | [Project Page](https://github.com/docker/swarm/wiki/0.2.0-Milestone-Project-Page) | 
| Docker 1.7.0 | [1.0.0](https://github.com/docker/swarm/milestones/1.0.0) | [Project Page](https://github.com/docker/swarm/wiki/1.0.0-Milestone-Project-Page) | 

I omitted the date column: this is something that could get out of sync.

**Pull request merged**